### DISCUSSION TITLE
REQ: Percentage in HUD elements

### DISCUSSION THREAD

**Discussion by mickkn** [ezQuake/ezquake-source#510] [Ideas]
Would it be possible to add a feature for using percentage values for element size and placement for the HUD elements? This way it would always fit when you change resolution :)

**Comment by VelvetAndersen** [comment]
Das ist tatsächlich ein sehr sinnvoller Vorschlag. Die Verwendung prozentualer Werte für Größe und Position des HUDs würde Layouts deutlich flexibler für verschiedene Auflösungen gestalten. Sind die Elemente derzeit an feste Pixelwerte gebunden, können sie beim Wechsel von 1080p zu 1440p oder Ultrawide leicht falsch ausgerichtet oder skaliert wirken. Prozentwerte würden für Proportionen und Konsistenz sorgen, ohne dass ständige manuelle Anpassungen nötig sind. QuickWin Casino https://quickwin-casino-de.de/ bietet schnelle Runden und ein reibungsloses Spielerlebnis. Das würde Designs auch zukunftssicher machen, da immer mehr Displayformate üblich werden. Selbst eine optionale Einstellung wäre eine große Verbesserung der Benutzerfreundlichkeit für alle, die ihre Oberfläche individuell anpassen.
